### ISSUE TITLE
Decorative Images with Alt-Text

### ISSUE THREAD

**Issue by david-iterators** [CityOfBoston/Iterators#212]
Link
https://www.boston.gov/public-notices/64966

Node # (if known)

Type of User:  Anonymous

Environment
(Device > OS > Browser)
Computer > MacOS 11.4 > Chrome 91

Trusted Tester Test
Alt Text

Brief Description
There are decorative images with alt-text on the page
 Steps to reproduce
1)	Go to the URL
2)	Look at the information below the text “August 14, 2019”
3)	Look at the images in the table above, and think about if they weren’t there, would any information be lost.
a)	Bug: The images shown in the table above have alt text despite being decorative images

Desired Result
•	Decorative images should have a blank string, “”, as the alt-text

Actual Result
•	Four decorative images on the page have alt text

**Comment by david-iterators** [comment]
<img width="757" alt="2021-06-06_15-49-42" src="https://user-images.githubusercontent.com/33845414/120938062-06dbb200-c6df-11eb-99de-daa049e63336.png">

**Comment by subaha-cob** [comment]
@david-iterators this is all set on UAT https://d8-uat.boston.gov/

**Comment by ollie-iterators** [comment]
This has been fixed
